.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,25 @@
+name: .NET
+
+on:
+  push:
+    branches: [ master, CIpipeline ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 6.0.x
```

Comment:
You can omit the version, and the version installed will be based off of the `global.json` file: https://github.com/actions/setup-dotnet#usage